styles/features.scss

```diff
@@ -92,6 +92,18 @@
       font-size: 1.2rem;
       color: $black10;
       max-width: none;
+      ul{
```

Comment:
you can use li list-style
https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type